manifests/etcdquorumguard/deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: etcd-quorum-guard
+  namespace: kube-system
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      k8s-app: etcd-quorum-guard
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: etcd-quorum-guard
+        k8s-app: etcd-quorum-guard
+    spec:
+      hostNetwork: true
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
```

Comment:
this must be `k8s-app` right? based on https://github.com/openshift/machine-config-operator/pull/613/files#diff-0c1db25c8965ca5514e8f958eb10f61dR20

Reply:
OK

Reply:
maybe this is why we get that not-found in tests 🤔

Reply:
That would be nice.  We'll find out soon enough...